File: sample-javascript-sequelize/src/model/PartyType.js
// model/PartyType.js
const Sequelize = require("sequelize");
const {sequelize} = require("../db");

exports.PartyType = sequelize.define(
  "PartyType",
  {
    id: {
      type: Sequelize.INTEGER,
      primaryKey: true,
      field: "party_type_id",
    },
    description: {
      type: Sequelize.STRING,
      field: "party_type_description",
    },
  },
  {tableName: "party_type", timestamps: false},
);

File: sample-java-jpa/src/main/java/hq/sombriks/sample/model/PartyType.java
package hq.sombriks.sample.model;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.Data;

@Data
@Entity
@Table(name = "party_type")
public class PartyType {

  public PartyType() {
  }

  public PartyType(int id) {
    this.id = id;
  }

  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  @Column(name = "party_type_id")
  private Integer id;

  @Column(name = "party_type_description")
  private String description;
  
}